specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNode.json

```diff
@@ -0,0 +1,949 @@
+    },
+    "DscReportResource": {
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "Gets or sets the ID of the resource."
+        },
+        "sourceInfo": {
+          "type": "string",
+          "description": "Gets or sets the source info of the resource."
+        },
+        "dependsOn": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DscReportResourceNavigation"
+          },
+          "description": "Gets or sets the Resource Navigation values for resources the resource depends on."
+        },
+        "moduleName": {
+          "type": "string",
+          "description": "Gets or sets the module name of the resource."
+        },
+        "moduleVersion": {
+          "type": "string",
+          "description": "Gets or sets the module version of the resource."
+        },
+        "resourceName": {
+          "type": "string",
+          "description": "Gets or sets the name of the resource."
+        },
+        "error": {
+          "type": "string",
+          "description": "Gets or sets the error of the resource."
+        },
+        "status": {
+          "type": "string",
+          "description": "Gets or sets the status of the resource."
+        },
+        "durationInSeconds": {
+          "type": "number",
+          "format": "double",
+          "description": "Gets or sets the duration in seconds for the resource."
+        },
+        "startDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the start date of the resource."
+        }
+      },
+      "description": "Definition of the DSC Report Resource."
+    },
+    "DscReportError": {
+      "properties": {
+        "errorSource": {
+          "type": "string",
+          "description": "Gets or sets the source of the error."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Gets or sets the resource ID which generated the error."
+        },
+        "errorCode": {
+          "type": "string",
+          "description": "Gets or sets the error code."
+        },
+        "errorMessage": {
+          "type": "string",
+          "description": "Gets or sets the error message."
+        },
+        "locale": {
+          "type": "string",
+          "description": "Gets or sets the locale of the error."
+        },
+        "errorDetails": {
+          "type": "string",
+          "description": "Gets or sets the error details."
+        }
+      },
+      "description": "Definition of the dsc node report error type."
+    },
+    "AgentRegistration": {
+      "properties": {
+        "dscMetaConfiguration": {
+          "type": "string",
+          "description": "Gets or sets the dsc meta configuration."
+        },
+        "endpoint": {
+          "type": "string",
+          "description": "Gets or sets the dsc server endpoint."
+        },
+        "keys": {
```

Comment:
keys should not be returned as part of the GET. Only via a POST.

Reply:
This is also legacy API